modules/javafx.graphics/src/main/native-glass/mac/GlassViewDelegate.m

```diff
@@ -1046,12 +1046,9 @@ - (void)startDrag:(NSDragOperation)operation withItems:(NSArray<NSDraggingItem*>
 
         if (image != nil)
         {
-            // select the center of the image as the drag origin
```

Comment:
The old comment about RT-17629 (https://bugs.openjdk.org/browse/JDK-8091965) does not apply anymore: after this PR, the offset, which comes from the Java layer, does adjust the drag image origin.

So I believe we can remove the whole comment as well (and possibly close RT-17629).